multinode_runfabtests.sh

```diff
@@ -37,6 +37,63 @@ run_dgram_pingpong_with_b()
     return $ret
 }
 
+run_rdm_tagged_bw_fork()
+{
+    SERVER_IP=$1
+    CLIENT_IP=$2
+    FORK_MODE=$3
+
+    if [ -z "${FORK_MODE}" ]; then
+        FORK_ENV=""
+        rdm_tagged_bw_cmd="${HOME}/libfabric/fabtests/install/bin/fi_rdm_tagged_bw -p efa -K -E"
+    elif [ ${FORK_MODE} = "RDMAV_FORK_SAFE" ]; then
+        FORK_ENV="RDMAV_FORK_SAFE=1"
+        rdm_tagged_bw_cmd="${HOME}/libfabric/fabtests/install/bin/fi_rdm_tagged_bw -v -p efa -K -E"
+    elif [ ${FORK_MODE} = "FI_EFA_FORK_SAFE" ]; then
+        FORK_ENV="FI_EFA_FORK_SAFE=1"
+        rdm_tagged_bw_cmd="${HOME}/libfabric/fabtests/install/bin/fi_rdm_tagged_bw -v -p efa -K -E"
+    fi
+
+    ssh -o ConnectTimeout=30 -o StrictHostKeyChecking=no ${SERVER_IP} ${FORK_ENV} ${rdm_tagged_bw_cmd} >& server.out &
```

Comment:
Do ${FORK_ENV} actually set the environment variable on remote instance?

Reply:
`FORK_ENV` sets the environment variable only during the duration when `fi_rdm_tagged_bw` is run. 

For example, if we call `run_rdm_tagged_bw_fork ${SERVER_IP} ${CLIENT_IP}` after `run_rdm_tagged_bw_fork ${SERVER_IP} ${CLIENT_IP} "FI_EFA_FORK_SAFE"`. No environment variables will be set.

Reply:
You are using ssh to execute the command remotely, right? and you want to set the environment variable on remote machine, not local machine. My question is did the command line actually set the environment variable on the remote machine? My experience has been that command line like the following:

`ssh user@host ENV=xxx programs` will not set `ENV` to `xxx` on the remote machine.

Reply:
I do not think there is a need to set environment variable on remote machine. By using the syntax `ssh user@host ENV=xxx programs`, we only temporary sets the `ENV` to `xxx` when running `programs`.

Reply:
But the application `fi_rdm_tagged_bw` is run on remote machine, and its behavior would be different depend on the environment variable, right? which is what we are testing?

Reply:
Never mind. I verified that ssh will pass env to remote machine.